src/building/building_state.h

```diff
@@ -5,8 +5,8 @@
 #include "core/buffer.h"
 
 #define BUILDING_STATE_ORIGINAL_BUFFER_SIZE 128
-#define BUILDING_STATE_TOURISM_BUFFER_SIZE 134
-#define BUILDING_STATE_CURRENT_BUFFER_SIZE 134
+#define BUILDING_STATE_CULTURE_BUFFER_SIZE 139
+#define BUILDING_STATE_CURRENT_BUFFER_SIZE 139
```

Comment:
The same here. You should keep the `#define BUILDING_STATE_TOURISM_BUFFER_SIZE 134` and then add the `#define BUILDING_STATE_CULTURE_BUFFER_SIZE 139` below it.